draft-ietf-quic-transport.md

```diff
@@ -3204,9 +3204,15 @@ response to further packets that it receives.
 A receiver MUST discard a newly unprotected packet unless it is certain that it
 has not processed another packet with the same packet number from the same
 packet number space. Duplicate suppression MUST happen after removing packet
-protection for the reasons described in Section 9.3 of {{QUIC-TLS}}. An
-efficient algorithm for duplicate suppression can be found in Section 3.4.3 of
-{{?RFC4303}}.
+protection for the reasons described in Section 9.3 of {{QUIC-TLS}}.
+
+Endpoints that track individual packets for the purposes of detecting duplicates
+might accumulate excessive state.  The data required for detecting duplicates
+can be limited by maintaining a minimum packet number below which all packets
+are immediately dropped.  In setting a minimum packet number endpoints might
+need to account for large variations in round trip time that could significantly
+delay packets, especially if a peer migrates to or probes a different network
+path; see {{migration}}.
```

Comment:
I think it might be a good idea to change this sentence to something like: _In setting a minimum packet number endpoints **have** to account for large variations in round trip time that could significantly delay packets, especially if a peer probes a different network path; see {migration}}._

The changes are use of "have to" rather than "might," and omitting the mention of migration.

Migration is not an issue, because when the peer migrates to a path with higher latency there would be no loss, and because when the peer migrates to a path with lower latency, the loss would be a one-time effect that can be recovered.

In contrast to that, for path migration to work, endpoints have to be capable of deduplicating packets with significant reordering. Without that, the peer cannot reliably probe a path while sending data.

I think this is essentially a requirement regardless of how we phrase it.

Reply:
I see how you might pick on probing exclusively.  I'll remove migration and just identify probing.

s/have/need means the same to me.  It's 6919 language of a different form :)

Reply:
Oh, I see.  I was thinking that you don't need to do this if you disable_active_migration.  Which is why the probing mention is important.